controllers/genreController.js

```diff
@@ -0,0 +1,25 @@
+const { Genre } = require('../models');
+
+/**
+ * Serves an array of movie genres
+ * @return  genres
```

Comment:
Include return type in comment for readability, and also the parameter types. 
e.g.:
```suggestion
/**
 * Serves an array of movie genres
 * @param {??} res the response // this should probably be more descriptive than I have it here
 * @return {Promise} genres
 */
```